rclcpp/include/rclcpp/subscription_factory.hpp

```diff
@@ -88,7 +89,7 @@ create_subscription_factory(
 
   SubscriptionFactory factory {
     // factory function that creates a MessageT specific SubscriptionT
-    [options, msg_mem_strat, any_subscription_callback](
+    [options, msg_mem_strat, any_subscription_callback, type_support](
```

Comment:
same thing. I think that's the place to extract the typesupport.

Reply:
same as create_subscription